scvi/inference/inference.py

```diff
@@ -47,7 +50,10 @@ def loss(self, tensors):
         return loss
 
     def on_epoch_begin(self):
-        self.kl_weight = self.kl if self.kl is not None else min(1, self.epoch / 400)  # self.n_epochs)
+        if self.n_epochs_kl_warmup is not None:
+            self.kl_weight = min(1, self.epoch / self.n_epochs_kl_warmup)
+        else:
+            self.kl_weight = 1.0
 
 
 class AdapterTrainer(UnsupervisedTrainer):
```

Comment:
@adamgayoso I believe the adapter trainer is only used to optimize the variational network at test time so we probably do not need to use any warmup on this